packages/beacon-node/src/db/buckets.ts

```diff
@@ -61,6 +61,9 @@ export enum Bucket {
   // 54 was for bestPartialLightClientUpdate, allocate a fresh one
   // lightClient_bestLightClientUpdate = 55, // SyncPeriod -> LightClientUpdate // DEPRECATED on v1.5.0
   lightClient_bestLightClientUpdate = 56, // SyncPeriod -> [Slot, LightClientUpdate]
+
+  allForks_dataColumnSidecars = 57, // FULU BeaconBlockRoot -> DataColumnSidecars
+  allForks_dataColumnSidecarsArchive = 58, // FULU BeaconBlockSlot -> DataColumnSidecars
```

Comment:
As per pattern for other keys, we use fork name where that bucket was introduced. 

```suggestion
  fulu_dataColumnSidecars = 57, // FULU BeaconBlockRoot -> DataColumnSidecars
  fulu_dataColumnSidecarsArchive = 58, // FULU BeaconBlockSlot -> DataColumnSidecars
```